### DISCUSSION TITLE
求助，小白甚至不会开启这个软件……

### DISCUSSION THREAD

**Discussion by I-am-newbie-be-patient-PLEASE** [Blinue/Magpie#350] [Q&A]
源代码已经打包下载，但是没看到能打开软件的.exe文件啊……
所以说这个文件要怎样才能启动呢？
谢谢大佬耐心解答

**Comment by Blinue** [comment]
在 https://github.com/Blinue/Magpie/releases 下载可执行文件

**Reply by I-am-newbie-be-patient-PLEASE** [reply]
懂了，下错了，把源代码给打包了……谢谢
